libensemble/tests/regression_tests/test_persistent_gp_multitask_ax.py

```diff
@@ -50,6 +50,7 @@ def run_simulation(H, persis_info, sim_specs, libE_info):
         z = 8
     elif task == "cheap_model":
         z = 1
+    print('in sim', task)
```

Comment:
Do you want this print to remain? (It's fine with me, but I wanted to point it out.)

Reply:
I did originally leave that print in by accident but then I thought it's actually kind of useful to see that it's doing both models.